python/pyspark/sql/column.py

```diff
@@ -303,8 +333,25 @@ def isin(self, *cols):
     desc = _unary_op("desc", "Returns a sort expression based on the"
                              " descending order of the given column name.")
 
-    isNull = _unary_op("isNull", "True if the current expression is null.")
-    isNotNull = _unary_op("isNotNull", "True if the current expression is not null.")
+    _isNull_doc = ''' True if the current expression is null. Often combined with 
```

Comment:
I just found a good reference in pep8

> For triple-quoted strings, always use double quote characters to be consistent with the docstring convention in PEP 257

https://www.python.org/dev/peps/pep-0008/#string-quotes